soccer/gameplay/tactics/defense.py

```diff
@@ -403,6 +413,10 @@ def recalculate_threat_shot(threat_index):
         # print("Unused handlers: " + str(unused_threat_handlers))
         # print("---------------------")
 
+        # If we have nothing to block, bail
+        if not threats:
+            return
+
         smart = False
         if not smart:
```

Comment:
Would you be ok with deleting the else block then? And by deleting it I mean creating an intro issue to delete it.